Run.ps1

```diff
@@ -1,49 +1,148 @@
-function Show-Menu
-{
-     param (
-           [string]$Title = 'NSB Monitoring Setup'
-     )
-
-     Clear-Host
-     Write-Host "================ $Title ================"
-     
-     Write-Host "1: Use existing SQL Server database."
-     Write-Host "2: Use LocalDB (requires LocalDB and elevated permissions)."
-     Write-Host "Q: Quit."
-     Write-Host
-}
+Set-Location $PSScriptRoot
 
-try
+function Write-Exception 
 {
-        Get-ChildItem -Recurse $PSScriptRoot | Unblock-File
-
-        Show-Menu
-
-        $input = Read-Host "Please make a selection and press <ENTER>"
-
-        switch ($input)
-        {
-                '1' {
-                        $args = [string]::Format('-NoExit -NoProfile -ExecutionPolicy Bypass -File "{0}\existingDB.ps1"', $PSScriptRoot)  
-                        Start-Process PowerShell.exe -ArgumentList $args -WorkingDirectory $PSScriptRoot -Verb runAs 
```

Comment:
This ensured that the users is not prompted for elevated permission startup of sc, monitoring and sp. Right now one has to click 3 times "Yes" to make the demo run.